lib/http2/hpack.c

```diff
@@ -256,7 +250,7 @@ static h2o_iovec_t *decode_string(h2o_mem_pool_t *pool, const uint8_t **src, con
     return ret;
 }
 
-static inline struct st_h2o_hpack_header_table_entry_t *header_table_get(h2o_hpack_header_table_t *table, size_t index)
+h2o_hpack_header_table_entry_t *h2o_hpack_header_table_get(h2o_hpack_header_table_t *table, size_t index)
```

Comment:
Assuming that all the structs used inside the function is defined in the headers, would it be possible to change this function to a static inline function declared and defined in http2_internal.h?

Please refer to other inline functions defined in http2_internal.h to how to define a global function as inline (in essence, declare as `static` and, define as `inline`).